packages/lodestar/src/db/repositories/block.ts

```diff
@@ -1,7 +1,7 @@
 import {IChainForkConfig} from "@chainsafe/lodestar-config";
 import {Bucket, Db, IDbMetrics, Repository} from "@chainsafe/lodestar-db";
-import {allForks, ssz} from "@chainsafe/lodestar-types";
-import {getSignedBlockTypeFromBytes} from "../../util/multifork";
```

Comment:
`getSignedBlockTypeFromBytes` should be in `util/multifork` not `lodestar-beacon-state-transition`